src/app/views/App.tsx

```diff
@@ -66,6 +68,7 @@ interface IAppState {
   selectedVerb: string;
   mobileScreen: boolean;
   hideDialog: boolean;
+  onToggleSidebar: boolean;
```

Comment:
This variable is named like an event. Changing it to `isSidebarToggled` may help clarify it's purpose

Reply:
Ah true, will do